commands/system-prune.ts

```diff
@@ -0,0 +1,9 @@
+import vscode = require('vscode');
+
+export function systemPrune() {
+
+    let terminal = vscode.window.createTerminal("docker system prune");
```

Comment:
i thought about this too, but then it opens a can of worms... do you start doing this for all commands?  the nice thing is that since it simply shells to the terminal, if you dont have 1.13 then the command will simply fail there.